nextflow/rmatsTurbo.nf

```diff
@@ -0,0 +1,232 @@
+nextflow.enable.dsl=2
+
+
+process rmats_prep_g1 {
+  tag "rmats_prep_g1"
+  label "mid_memory"
+  publishDir path: "${params.publishDir}/read_outcomes", mode: 'copy', pattern: "prep_*_read_outcomes_by_bam.txt"
+
+  input:
+    tuple val(bam_name), file(bam)
+    each file(gtf)
+    val(group)
+
+  output:
+    path "outfd/*.rmats", emit: rmat
+    path "prep_${bam_id}_read_outcomes_by_bam.txt", emit: rob
+    val "${bam_name}", emit: bam_name
```

Comment:
Is this `bam_name` output used? It looks like later the code just uses the input channel

Reply:
Not used. I removed it.